src/main/java/org/folio/config/ApplicationConfig.java

```diff
@@ -105,6 +109,36 @@
 @ComponentScan({ "org.folio" })
 public class ApplicationConfig {
 
+  private static final Logger LOGGER = LogManager.getLogger();
+
+  @Value("${KAFKA_HOST:kafka}")
+  private String kafkaHost;
+  @Value("${KAFKA_PORT:9092}")
+  private String kafkaPort;
+  @Value("${OKAPI_URL:http://okapi:9130}")
```

Comment:
nitpick: I think this should default to `http://localhost:9130`

Reply:
the default values used for the container in vagrant

Reply:
sounds good